Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ceph"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
This shouldn't be changed until we're ready to release a new version, at which point it should be a major version change